rocks/neutron-server/rockcraft.yaml

```diff
@@ -41,5 +41,6 @@ parts:
       - python3-ironic-neutron-agent
       - python3-neutron-dynamic-routing
       - python3-neutron-vpnaas
+      - python3-networking-generic-switch
```

Comment:
package does not exist in noble [1]
Can you use pip for the time being with version 7.3.0 [2]
Please add a comment as well with TODO so that we will upload package to noble

[1] https://launchpad.net/ubuntu/+source/networking-generic-switch
[2] https://docs.openstack.org/releasenotes/networking-generic-switch/2024.1.html